test/e2e/testnet/testnet.go

```diff
@@ -61,11 +66,11 @@ func (t *Testnet) Knuu() *knuu.Knuu {
 	return t.knuu
 }
 
-func (t *Testnet) NewPreloader() (*preloader.Preloader, error) {
+func (t *Testnet) NewPreloader(name string) (*preloader.Preloader, error) {
```

Comment:
Nit: could we just use the name of the testnet here and not need to add the argument?

Reply:
applied